Dockerfile

```diff
@@ -1,8 +1,12 @@
-FROM python:3.11
+FROM python:3.10-slim
 
 WORKDIR /app
-COPY scripts/ /app
 
-RUN pip install -r requirements.txt
+COPY requirements.txt /app/requirements.txt
```

Comment:
This change is done elsewhere (in different PRs) already. Search for the PR number(s) and add a comment. Remove from this PR.